packages/fxa-auth-server/lib/routes/linked-accounts.ts

```diff
@@ -139,6 +142,20 @@ export class LinkedAccountHandler {
       sessionToken: sessionToken.data,
     };
   }
+
+  async unlinkAccount(request: AuthRequest) {
+    if (!this.googleAuthClient) {
+      throw error.thirdPartyAccountError();
+    }
+    const uid = request.auth.credentials.uid;
+    if ((request.payload as any).provider.toLowerCase() === 'google') {
```

Comment:
Could the any cast be avoided? Maybe payload.provider can be added as an optional property in the AuthRequest interface. Alternatively, a new interface could be extended from AuthRequest to support this request format.

Reply:
I'll leave this as is for now